README.md

```
@@ -1,112 +1,34 @@
-<a href="https://x.com/RevoFarmers">
-  <img src="https://img.shields.io/twitter/follow/RevoFarmers?style=social"/>
-</a>
-<img src="https://img.shields.io/github/stars/Crypto-Jaguars/Revolutionary_Farmers?style=social"/>
+<a href="https://x.com/RevoFarmers"> <img src="https://img.shields.io/twitter/follow/RevoFarmers?style=social"/> </a> <img src="https://img.shields.io/github/stars/Crypto-Jaguars/Revolutionary_Farmers?style=social"/>
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)